source/extensions/filters/http/composite/config.cc

```diff
@@ -10,13 +10,14 @@ namespace Extensions {
 namespace HttpFilters {
 namespace Composite {
 
-Http::FilterFactoryCb CompositeFilterFactory::createFilterFactoryFromProtoTyped(
+absl::StatusOr<Http::FilterFactoryCb> CompositeFilterFactory::createFilterFactoryFromProtoTyped(
```

Comment:
I think this PR is missing some changes in [createActionFactoryCb](https://github.com/envoyproxy/envoy/blob/b68af06271838f62abf80ac6a2f84b0b5c413d68/source/extensions/filters/http/composite/action.cc#L15)() function composite/action.cc. That is place to decide the nature of delegated filter. 

What this PR currently has : upstream match_delegate/composite delegates **downstream filter**, while what we want is delegating to **upstream filter**.

Reply:
Good catch! Update the change. PTAL.